api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -397,61 +397,63 @@ def test_multiplexed_metadata(self):
         self.assertEqual(publication2.doi, "{doi 2}")
         self.assertEqual(publication2.citation, "{formatted citation 2}")
 
-        expected_keys = [
-            "scpca_sample_id",
-            "scpca_library_id",
-            "scpca_project_id",
-            "technology",
-            "seq_unit",
-            "total_reads",
-            "mapped_reads",
-            "genome_assembly",
-            "mapping_index",
-            "date_processed",
-            "workflow",
-            "workflow_version",
-            "workflow_commit",
-            "diagnosis",
-            "subdiagnosis",
-            "pi_name",
-            "project_title",
-            "disease_timing",
-            "age_at_diagnosis",
-            "sex",
-            "tissue_location",
-            "participant_id",
-            "submitter",
-            "submitter_id",
-            "alevin_fry_version",
-            "cell_filtering_method",
-            "demux_method",
-            "demux_samples",
-            "development_stage_ontology_term_id",
-            "disease_ontology_term_id",
-            "droplet_filtering_method",
-            "filtered_cell_count",  # with non-multiplexed
-            "filtered_cells",
-            "has_cellhash",
-            "includes_anndata",
-            "is_cell_line",
-            "is_multiplexed",
-            "is_xenograft",
-            "min_gene_cutoff",
-            "normalization_method",
-            "organism",
-            "organism_ontology_id",
-            "prob_compromised_cutoff",
-            "processed_cells",
-            "salmon_version",
-            "sample_cell_count_estimate",  # with non-multiplexed
-            "sample_cell_estimates",
-            "self_reported_ethnicity_ontology_term_id",
-            "sex_ontology_term_id",
-            "tissue_ontology_term_id",
-            "transcript_type",
-            "unfiltered_cells",
-            "WHO_grade",
-        ]
+        expected_keys = utils.get_sorted_field_names(
```

Comment:
- break this up into `expected_project_keys` and `expected_sample_keys`
- remove the explicit sorting everywhere it was added as values we test against here should just be hard coded to remove hidden logic